convert2rhel/actions/post_conversion/hostmetering.py

```diff
@@ -0,0 +1,235 @@
+                description="Host metering is not needed on the system.",
+            )
+            return False
+
+        logger.info("Installing host-metering packages.")
+        output, ret_install = call_yum_cmd("install", ["host-metering"])
+        if ret_install:
+            logger.warning("Failed to install host-metering rpms.")
+            # TODO Consider this when setting this action as dependency of other actions
+            # Fail in this step can result in incomplete conversion - even if it would be
+            # enough to re-run the install and config steps.
+            self.set_result(
+                level="ERROR",
+                id="FAILED_INSTALL_HOST_METERING",
+                title="Failed to install packages.",
+                description="Failed to install host-metering rpm.",
+                diagnosis="`yum install host-metering` command returned {ret_install} with message {output}".format(
+                    ret_install=ret_install, output=output
+                ),
+                remediations="You can try install and set up the host metering"
+                " manually using following commands:\n"
+                "   yum install host-metering\n"
+                "   systemctl enable host-metering.service\n"
+                "   systemctl start host-metering.service",
+            )
+            return False
+
+        command, error_message = self._enable_host_metering_service()
+        # If there is any failure, the failed command would be present
+        if any(command):
+            self.set_result(
+                level="ERROR",
+                id="FAILED_CONFIGURE_HOST_METERING",
+                title="Failed to configure host metering.",
+                description="Failed to enable and start host metering service.",
+                diagnosis="Command {command} failed with {error_message}".format(
+                    command=command, error_message=error_message
+                ),
+                remediations="You can try set up the host metering"
+                " manually using following commands:\n"
+                "   systemctl enable host-metering.service\n"
+                "   systemctl start host-metering.service",
+            )
+            return False
+
+        service_running = systeminfo.is_systemd_managed_service_running("host-metering.service")
+
+        if not service_running:
+            logger.critical_no_exit("host-metering service is not running.")
+            self.set_result(
+                level="ERROR",
+                id="HOST_METERING_NOT_RUNNING",
+                title="Host metering service is not running.",
+                description="host-metering.service is not running.",
+                remediations="You can try to start the service manually"
+                " by running following command:\n"
+                "   systemctl start host-metering.service",
+            )
+
+        return service_running
+
+    def _check_env_var_correct(self):
```

Comment:
For reviewers:

I had to split the code a bit since there were a lot of `return` statements.